[
    {
        "content": "<p><a href=\"https://github.com/leanprover-community/mathlib3port\">https://github.com/leanprover-community/mathlib3port</a> was last updated 3 days ago<br>\ncc <span class=\"user-mention\" data-user-id=\"110049\">@Mario Carneiro</span> <span class=\"user-mention\" data-user-id=\"110043\">@Gabriel Ebner</span></p>",
        "id": 321763802,
        "sender_full_name": "Johan Commelin",
        "timestamp": 1673938033
    },
    {
        "content": "<p>It seems to be timing out <a href=\"https://github.com/leanprover-community/mathport/actions/runs/3935945845/jobs/6732060407\">https://github.com/leanprover-community/mathport/actions/runs/3935945845/jobs/6732060407</a></p>",
        "id": 321764069,
        "sender_full_name": "Johan Commelin",
        "timestamp": 1673938224
    },
    {
        "content": "<p>Running the <code>mathport</code> action used to take about 45mins, then suddenly it bumped to ~2hrs. So I guess it's just being cancelled by the next run, all the time.</p>",
        "id": 321764337,
        "sender_full_name": "Johan Commelin",
        "timestamp": 1673938423
    },
    {
        "content": "<p>Unreachable code has been reached: <a href=\"https://github.com/leanprover-community/mathport/actions/runs/3925923824/jobs/6711163868\">https://github.com/leanprover-community/mathport/actions/runs/3925923824/jobs/6711163868</a></p>",
        "id": 321764442,
        "sender_full_name": "Johan Commelin",
        "timestamp": 1673938495
    },
    {
        "content": "<p>And in the end it always seems to crash with<br>\n<code>Failed to port { package := \"Mathbin\", mod3 := `data.polynomial.mirror }</code></p>",
        "id": 321764485,
        "sender_full_name": "Johan Commelin",
        "timestamp": 1673938549
    },
    {
        "content": "<p>But <code>data.polynomial.mirror</code> hasn't been touched since Nov 25. So it can't be related to a change in that file.</p>",
        "id": 321764610,
        "sender_full_name": "Johan Commelin",
        "timestamp": 1673938628
    },
    {
        "content": "<p><a href=\"#narrow/stream/287929-mathlib4/topic/mathport.20fails.20after.20mathlib4.20update/near/321270108\">the previous thread</a></p>",
        "id": 321807505,
        "sender_full_name": "Eric Wieser",
        "timestamp": 1673954775
    },
    {
        "content": "<p>I'm getting old... I had already forgotten about that thread <span aria-label=\"oops\" class=\"emoji emoji-1f643\" role=\"img\" title=\"oops\">:oops:</span> <span aria-label=\"see no evil\" class=\"emoji emoji-1f648\" role=\"img\" title=\"see no evil\">:see_no_evil:</span></p>",
        "id": 321810502,
        "sender_full_name": "Johan Commelin",
        "timestamp": 1673955739
    },
    {
        "content": "<p>The updates have resumed.</p>",
        "id": 321972295,
        "sender_full_name": "Gabriel Ebner",
        "timestamp": 1674016916
    },
    {
        "content": "<p>Wow, that's awesome!</p>",
        "id": 321973360,
        "sender_full_name": "Johan Commelin",
        "timestamp": 1674017773
    },
    {
        "content": "<p>Thank you!</p>",
        "id": 322000591,
        "sender_full_name": "Kevin Buzzard",
        "timestamp": 1674032335
    }
]